### PR TITLE
mcobbett maven uses common code

### PR DESCRIPTION
**Pull request by techcobweb**
- maven plugin to use common code.
- avoid junit vulnerability

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-maven-pr-115-nfcrr for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-maven-pr-115-fsfjq for details. If you are unable to do so, please contact a member of the Galasa team.

File: galasa-maven-plugin/pom.xml
Comment (jadecarino) [inline galasa-maven-plugin/pom.xml:10]: 0.33.0

File: galasa-maven-plugin/pom.xml
Comment (jadecarino) [inline galasa-maven-plugin/pom.xml:100]: Versions need adjusting

File: galasa-maven-plugin/README.md
Comment (jadecarino) [inline galasa-maven-plugin/README.md:17]: These headings haven't rendered for some reason

File: galasa-maven-plugin/README.md
Comment (jadecarino) [inline galasa-maven-plugin/README.md:66]: Unsure what the difference between these two are, I was meant to ask you last week. Can we discuss?

**Review by jadecarino — CHANGES_REQUESTED** [review]
Happy to approve once the versions have been merged and comment about bundle vs deploy addressed

File: galasa-maven-plugin/README.md
Comment (jadecarino) [reply galasa-maven-plugin/README.md:66]: Is the description meant to be different on these two lines?

File: galasa-maven-plugin/pom.xml
Comment (techcobweb) [reply galasa-maven-plugin/pom.xml:100]: done.

File: galasa-maven-plugin/README.md
Comment (techcobweb) [reply galasa-maven-plugin/README.md:66]: Explained more in the readme.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-maven-pr-115-4cqk9 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-maven-pr-115-ddrnx for details. If you are unable to do so, please contact a member of the Galasa team.

File: galasa-maven-plugin/pom.xml
Comment (techcobweb) [reply galasa-maven-plugin/pom.xml]: Fixed.

File: galasa-maven-plugin/pom.xml
Comment (techcobweb) [reply galasa-maven-plugin/pom.xml]: fixed.

File: galasa-maven-plugin/README.md
Comment (techcobweb) [reply galasa-maven-plugin/README.md:17]: Fixed

File: galasa-maven-plugin/pom.xml
Comment (techcobweb) [reply galasa-maven-plugin/pom.xml]: Can't mark this conversation as resolved.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-maven-pr-115-jw8bh for details. If you are unable to do so, please contact a member of the Galasa team.

**Review by jadecarino — APPROVED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-maven-pr-115-nsrmk for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-maven-pr-115-48bzh for details. If you are unable to do so, please contact a member of the Galasa team.

**Pull request merged**
